app/summit-2016/data/schedule.json

```diff
@@ -1,162 +1,142 @@
 [
   {
-    "date":"September 14th",
+    "date":"October 17th",
     "day":"Monday",
     "sessions":[
       {
-        "name":"Code Labs and Pre-registration",
-        "speaker":"",
-        "time":"Monday 5pm - 9pm",
-        "desc":"Polymer code labs provide a guided, tutorial, hands-on coding experience. Learn and build elements and applications with the Polymer Team. **BYOL (Bring Your Own Laptop)**"
-      }
-    ]
-  },
-  {
-    "date":"September 15th",
-    "day":"Tuesday",
-    "sessions":[
+        "name":"Opening Keynote",
+        "speaker":"Taylor Savage",
+        "desc":"Welcome to the second Polymer Summit! We'll kick off with a whirlwind tour of what's new in the world of Web Components and the Polymer Project."
+      },
       {
-        "name":"Registration Opens",
-        "time":"Tuesday @ 7:30am"
+        "name": "The Way of the Web - ING",
```

Comment:
Throughout: In the docs, we always use em dashes with no space: "The Way of the Web-ING" In the case of ING, that looks a little weird... Could use "The Way of the Web: ING"?